apps/auction2/client.js

```diff
@@ -1,8 +1,24 @@
 import Auction from '../../models/auction.coffee'
+import CurrentUser from '../../models/current_user.coffee'
 import ClockView from '../../components/clock/view.coffee'
 import { data as sd } from 'sharify'
 import _ from 'underscore'
+import MyActiveBids from '../../components/my_active_bids/view.coffee'
+
+const myActiveBidsTemplate = require('./templates/my_active_bids.jade')
```

Comment:
🤔  I assume this is b/c Browserify was having trouble parsing `import`s for jade files? I wonder if we just need to swap the order of the transform.s